pages/instructorAssessmentGroups/instructorAssessmentGroups.js

```diff
@@ -92,10 +104,10 @@ router.post('/', function(req, res, next) {
             if (ERR(err, next)) return;
             res.redirect(req.originalUrl);
         });
-    } else if (req.body.__action == 'addGroup') {
+    } else if (req.body.__action == 'add_group') {
         const assessment_id = res.locals.assessment.id;
-        const groupname = req.body.groupname;
-        if (String(groupname).length < 1) {
+        const group_name = req.body.group_name;
+        if (String(group_name).length < 1) {
```

Comment:
If `group_name` is `undefined` then I think this will change it to the string `"undefined"` and pass this check?